h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -910,4 +928,112 @@ private void cleanUpModelsCVPreds() {
         model.deleteCrossValidationPreds();
     }
   }
+
+
+  protected AutoBuffer writeAutoML(AutoBuffer ab) {
+    try (PersistenceContext pc = PersistenceContext.begin()) {
+      ab.put(_key);
+      ab.put(_buildSpec);
+      ab.putKey(_leaderboard._key);
+      ab.putKey(_eventLog._key);
+//      ab.putKey(_trainingFrame._key);
+      for (Model m : leaderboard().getModels()) {
+        PersistenceContext.putKey(ab, m._key);
+        if (_buildSpec.build_control.keep_cross_validation_predictions)
+          for (Key k : m._output._cross_validation_predictions)
+            PersistenceContext.putKey(ab,k);
+        if (_buildSpec.build_control.keep_cross_validation_models)
+          for (Key k : m._output._cross_validation_models)
+            PersistenceContext.putKey(ab,k);
+        if (_buildSpec.build_control.keep_cross_validation_fold_assignment)
+          PersistenceContext.putKey(ab,m._output._cross_validation_fold_assignment_frame_id);
+//        if (m instanceof StackedEnsembleModel)
+//          PersistenceContext.putKey(ab, ((StackedEnsembleModel)m)._output._metalearner._parms._train);
```

Comment:
The commented lines when uncommented enable us to use `.get_leaderboard("ALL")` and get non-NA values in `predict_time_per_row_ms`.